alpaca/entities_test.go

```diff
@@ -0,0 +1,35 @@
+package alpaca
+
+import (
+	"encoding/json"
+	"testing"
+
+	"cloud.google.com/go/civil"
+	"github.com/shopspring/decimal"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNonTradeActivities(t *testing.T) {
```

Comment:
First of all: thank you for adding a unit test! But, could you please create a test similar to the other one in rest_test.go? Other than keeping the consistency in the repo it would also be useful to test the whole `GetAccountActivities` method, not only the unmarshaling.